### PR TITLE
Refresh page after full check

### PR DESCRIPTION
**Pull request by Zlopez**
Signed-off-by: Michal Konečný <mkonecny@redhat.com>

### CODE REVIEW

File: anitya/templates/project.html [hunk 1]
```diff
@@ -281,6 +281,7 @@ <h3 class="panel-title">
               output = res.error;
             }
             $('#version_info').html("Latest versions: " + output);
+            window.location.reload(false);
```
Comment (cverna) [inline anitya/templates/project.html]: Just out curiosity why is this needed ?

Comment (Zlopez) [reply anitya/templates/project.html]: This is a button on the page. If you click on it, it will do a regular check for new versions. This could change project in the database, but the page will still have old information. It's confusing for the user that used this button.
